### DISCUSSION TITLE
https://johanneskonings.dev/notes/2024-07-12-misc/

### DISCUSSION THREAD

**Discussion by giscus** [JohannesKonings/JohannesKonings.github.io#28] [Note comments]
# https://johanneskonings.dev/notes/2024-07-12-misc/

https://johanneskonings.dev/notes/2024-07-12-misc/

**Comment by JohannesKonings** [comment]
https://github.com/aws-samples/alarm-context-tool

**Comment by JohannesKonings** [comment]
https://www.npmjs.com/package/@arethetypeswrong/cli

**Comment by JohannesKonings** [comment]
https://github.com/line/tsr
